### ISSUE TITLE
LXD storage driver

### ISSUE THREAD

**Issue by nealef** [anup-kodlekere/gaplib#25]
Using the dir storage driver appears to be the worst option. Check https://documentation.ubuntu.com/lxd/en/stable-5.0/reference/storage_drivers/. I think we should explore lvm thin pools to optimise start up etc.

**Comment by anup-kodlekere** [comment]
Yeah, we did some research over this a while ago and decided (not conclusively, we're still open to other options) to use LVM (with a ZFS storage driver) pool. We haven't worked on tweaking / fine-tuning the configs for our workloads so that is one thing which could show better results.
